bin/watch-packages-and-rebuild.js

```diff
@@ -7,14 +7,24 @@ const promiseExecFile = util.promisify( require( 'child_process' ).execFile );
 const debounce = require( 'lodash/debounce' );
 const debouncedProcessRebuildQueue = debounce( processRebuildQueue, 100 );
 
-const ignoredPathPieces = [ 'dist', 'test', 'tests' ];
-const ignoredPathRegexps = [ /packages\/calypso-color-schemes\/js\/index\.js/ ];
-const ignoredExtensions = [ '.css', '.md', '.d.ts' ];
-
+const ignoredPathPieces = [ 'node_modules', 'dist', 'test', 'tests' ].join( '|' );
+const ignoredExtensions = [ 'html', 'css', 'md', 'd.ts', 'png', 'svg' ]
+	.map( ( extension ) => '\\.' + extension ) // Need to escape the dot for regex
+	.join( '|' );
+const ignoredFiles = [
+	'^\\.|/\\.', // Ignores path patterns that start with a dot or have slash dot in sequence
+	'Makefile',
+	'LICENSE',
+	'Dockerfile',
+	'packages/calypso-color-schemes/js/index.js',
```

Comment:
I think this should use `path.join` as well. To make sure everything works on Windows. 

```suggestion
	path.join('packages', 'calypso-color-schemes', 'js', 'index.js'),
```

Not sure how the regex will behave then though.

Reply:
Thanks for pointing this out! I will follow up with a PR after testing it in my windows env!